fe/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -340,6 +340,9 @@
     @ConfField(mutable = true, masterOnly = true)
     public static int max_layout_length_per_row = 100000; // 100k
 
+    public static final int DEFAULT_DUP_KEYS_COUNT = 3;
+    public static final int DEFAULT_DUP_KEYS_BYTES = 36;
```

Comment:
Can you make these name consistent with others? And add some comment for them.
And is it your intention that keep them without ConfField annotation?

Reply:
In my opinion, the default value about keys may not be changed. The reason is that user could specify keys of table in create table stmt.
The name of final filed should use the UPPER CASE, right?